### DISCUSSION TITLE
How to shift the relative permeability curve

### DISCUSSION THREAD

**Discussion by ma-sadeghi** [PMEAL/OpenPNM#2661] [Q&A]
Question from Nikolina Susilovic by email:

> Could you please instruct me on which parameters to change in the [relative permeability](https://openpnm.org/examples/applications/relative_permeability.html) example, so that the relative permeability curve crossing point can be shifted left or right (attachment).
Additionally, with which parameters can I delay the wetting phase flow?

<img src="https://user-images.githubusercontent.com/14086031/210573157-17bc5753-65e7-4473-8caa-62b8e4c0a0c6.png" width=70%>

**Comment by jgostick** [comment]
I think the strongest fact in these curves is the pore size distribution and their spatial arrangement.  Consider the non-wetting phase: If the distribution is more narrow, then the pores which are invaded first (on the left side of the graph) then the permeability will be higher, thereby raising the blue line.  This will cause the intersection to move to the left.  I'm just making an educated guess though, since it's been quite a long time since I did serious rel perm curves.
